api/data_explorer/controllers/export_url_controller.py

```diff
@@ -69,7 +71,7 @@ def _check_preconditions():
         raise BadRequest(error_msg)
 
 
-def _get_entities_dict():
+def _get_entities_dict(cohortName, query):
```

Comment:
[Python style](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#316-naming) is cohort_name